pom.xml

```diff
@@ -81,7 +81,7 @@
     <seam.version>2.3.1.Final</seam.version>
     <slf4j.version>1.7.12</slf4j.version>
     <gwteventservice.version>1.2.1</gwteventservice.version>
-    <okapi.version>0.22</okapi.version>
+    <okapi.version>0.29-SNAPSHOT</okapi.version>
```

Comment:
Unfortunately not. The reason for using 29-snapshot is that they fixed a bug for me related to plurals.